libraries/monitor/src/builtins.js

```diff
@@ -166,9 +166,9 @@ MonitorManager.registerMetric('httpRequestsTotal', {
   description: 'Total number of HTTP requests',
   labels: {
     method: 'HTTP method',
-    path: 'HTTP path',
+    name: 'API method name',
```

Comment:
those metrics wasn't exposed before, so change of label name is not really breaking anything